services-js/commissions-search/src/FacetList.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from "react";
+import "./FacetList.css";
+
+import gql from "graphql-tag";
+import { Query } from "react-apollo";
+
+class FacetList extends Component {
+  render() {
+    const GET_AREAS = gql`
+      {
+        policyTypes {
+          id
+          name
+        }
+      }
+    `;
+
+    function Area(props) {
```

Comment:
Move to a `renderArea` method to keep `render` from getting so big.

Reply:
done